src/utils/helpers.js

```diff
@@ -0,0 +1,8 @@
+export const getDate = () => {
+	const weekday = new Date().toLocaleDateString("en-US", { weekday: "long" });
+	const numericDay = new Date().toLocaleDateString("en-US", { day: "numeric" });
+	const month = new Date().toLocaleDateString("en-US", { month: "long" });
+	const currentDate = `${weekday}, ${numericDay} ${month}`;
+
+	return currentDate;
```

Comment:
A czy toLocaleDateString nie wystarczy? W sensie, odpowiedni locale i będziesz miał to samo zamiast pisania helpera :)

Reply:
Tutaj zależy mi na takiej a nie innej kolejności wyświetlania treści i nie wiem chyba do końca jak to zrobić bez helpera